scenarios/10_Siege_of_Haeltin.cfg

```diff
@@ -258,9 +277,72 @@
             [/have_unit]
             [then]
                 {MSG_Reumario _"We really should sacrifice an ox to God. There are so many of them; without God's help we won't be able to defeat them."}
+                {MSGW_Yahyazad _"We don't even have any oxen, though."}
+                {MSG_Reumario _"Well, something similar, then."}
+                [unit]
+                    type=Woodland Boar
+                    side=5
+                    id=sacrificial_boar
+                    x,y=4,4
+                    animate=yes
+                    [ai]
+                        [micro_ai]
+                            ai_type=forest_animals
+                            tusker_type=Woodland Boar
+                            [filter_location]
+                                terrain=*^F*,Gg,Hh,Re,Ss
+                            [/filter_location]
+                        [/micro_ai]
+                    [/ai]
+                [/unit]
+                {MSG_Reumario _"...like that boar! Let me go capture it, sacrifice it, and then surely God's favor will be upon us!"}
```

Comment:
```suggestion
                {MSG_Reumario _"...like that boar! Let me go capture it and sacrifice it. Then perhaps God's favor will be upon us."}
```

Reply:
Eh, I don't think it's like Reumario to hedge like that; he seems more certain in his faith than that (he was the one who was intent on doing the sacrifice in the first place, after all)

Reply:
Really? I thought of him more as a faith somewhat shaken kind of guy considering all the stuff he had (and will have) to go through. Akladians are more the ones full of confidence in "God". I'll leave it to you though.

Reply:
> Really? I thought of him more as a faith somewhat shaken kind of guy considering all the stuff he had (and will have) to go through. Akladians are more the ones full of confidence in "God". I'll leave it to you though.

Well yeah, Reumario *is* an Akladian, so...